lib/sorcery/model/submodules/brute_force_protection.rb

```diff
@@ -14,6 +14,8 @@ def self.included(base)
                           :consecutive_login_retries_amount_limit,    # how many failed logins allowed.
                           :login_lock_time_period,                    # how long the user should be banned.
                           # in seconds. 0 for permanent.
+                          :limitless_counting_failed_login,           # When true, continue to count failed login
```

Comment:
I don't think another config option is needed here. Is there a reason why someone would want to disable this option?